frontend/src/ducks/ledgers.ts

```diff
@@ -141,6 +143,35 @@ export const fetchLedgersTransactionsHistoryAction = createAsyncThunk<
   },
 );
 
+export const fetchLedgerOperations = createAsyncThunk<
+  FetchLedgerOperationsResponse[],
+  void,
+  { rejectValue: RejectMessage; state: RootState }
+>("ledgers/fetchLedgerOperations", async (_, { rejectWithValue }) => {
+  try {
+    const historyFilter = ledgerTransactionHistoryConfig["30D"];
+    const response = await fetch(
+      `/api/ledgers${historyFilter.endpointPrefix}${
+        networkConfig[Network.MAINNET].ledgerTransactionsHistorySuffix
+      }`,
+    );
+
+    const operations = await response.json();
+
+    const result = operations.data.map((operation: LedgerModuleItem) => ({
+      date: parseDateFromFormat(
```

Comment:
this `fetchLedgerOperations` call is failing with error `Invalid time value`, probably the parsing is incorrect somehow

Reply:
😬 sorry I didn't see that